src/docs/development/add-to-app/index.md

```diff
@@ -35,40 +35,40 @@ Flutter into your own app.
   </div>
 </div>
 
-As of Flutter release v1.12, add-to-app is supported for the basic scenario of
-integrating one fullscreen Flutter instance at a time per app. It presently has
+As of Flutter v1.12, add-to-app is supported for the basic scenario of
+integrating one full-screen Flutter instance at a time per app. It currently has
 the _**following limitations**_:
 
-- Running multiple Flutter instances or in partial screen views may have
-  undefined behaviors.
-- Using Flutter in background mode is still WIP.
+- Running multiple Flutter instances or running in partial screen views may have
+  undefined behavior.
+- Using Flutter in background mode is still a WIP.
 - Packing a Flutter library into another sharable library or packing multiple
-  Flutter libraries into an application is not currently supported.
+  Flutter libraries into an application isn't supported.
```

Comment:
I still wanted to provide a small spark of hope to readers (since we might work on it soon). 'isn't' sounds a bit permanent.

Reply:
I see your point!